transit_odp/data_quality/pti/functions.py

```diff
@@ -232,14 +232,31 @@ def has_unregistered_service_codes(context, services):
     service_code_list = []
     service = services[0]
     ns = {"x": service.nsmap.get(None)}
+    flexible_service_code = None
 
     service_list = service.xpath("x:Service", namespaces=ns)
     for service in service_list:
+        flexible = service.xpath("x:ServiceClassification/x:Flexible", namespaces=ns)
+        flexible_service = service.xpath("x:FlexibleService", namespaces=ns)
```

Comment:
Is this required - If <Flexible /> element exists then its a flexible service

Reply:
@alvak1 I need these to check if the unregistered service code is a flexible service or not (as it could be a standard service code) and save the service code for comparison later.

Reply:
as discussed we could avoid like 239 and 240 as this function is called only if service if a flexible service